poly-commitment/benches/poly_comm.rs

```diff
@@ -0,0 +1,112 @@
+use ark_ec::{
+    short_weierstrass::{Affine, SWCurveConfig},
+    AffineRepr,
+};
+use ark_ff::{PrimeField, UniformRand};
+use criterion::{black_box, criterion_group, criterion_main, BatchSize, BenchmarkId, Criterion};
+use mina_curves::pasta::{Pallas, Vesta};
+use poly_commitment::PolyComm;
+use rand_core::{CryptoRng, RngCore};
+
+fn helper_generate_random_elliptic_curve_point<RNG, P>(rng: &mut RNG) -> Affine<P>
+where
+    P::BaseField: PrimeField,
+    RNG: RngCore + CryptoRng,
+    P: SWCurveConfig,
+{
+    let p1_x = P::BaseField::rand(rng);
```

Comment:
You could have just generated a scalar and scaled by it probably? I'm also thinking there's /likely/ a way to generate random points right away, no?

Reply:
Huhu. YES!

Reply:
"Pourquoi faire simple quand on peut faire compliqué ?"

Reply:
See https://github.com/o1-labs/proof-systems/pull/2612